jina/executors/encoders/image/customtfkeras.py

```diff
@@ -0,0 +1,48 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+
+from ..frameworks import BaseCVTFEncoder
+from ...decorators import batching, as_ndarray
+
+
+
+class CustomImageKerasEncoder(BaseCVTFEncoder):
```

Comment:
The `CustomImageKerasEncoder` can be moved into `encoders/image/tfkeras.py`.

Reply:
Plus, it is better to have a unit test. It will help us to make sure everything actually works out of box. Here is a sample codes which should go into `jina/tests/executors/encoders/image/test_tfkeras.py`.

```
class CustomizedKerasImageEncoderTestCase(ImageTestCase):
    def _get_encoder(self, metas):
        self.target_output_dim = 128
        self.input_dim = 64
        import tensorflow.keras as keras
        model = keras.Sequential([
            keras.layers.Conv2D(
                self.target_output_dim,
                self.input_dim,
                activation='relu',
                input_shape=(self.input_dim, self.input_dim, 3),
                name='conv'),
            keras.layers.Reshape(
                (self.target_output_dim, ),
                name='reshape')])
        model_path = './tmp_keras_model'
        model.save(model_path)
        self.add_tmpfile(model_path)
        return CustomImageKerasEncoder(model_path=model_path, layer_name='reshape', channel_axis=1, metas=metas)
```

Reply:
Okay. If you don't mind me asking why is `encoders/image/customtorchvision.py` as a separate file but the `CustomImageKerasEncode` be included in `encoders/image/tfkeras.py` ?
Please excuse my inexperience, I am still learning pytest. I'll execute the tests soon.

Reply:
Good point! We're do refactoring on that.